queries/php_only/highlights.scm

```diff
@@ -153,6 +153,14 @@
 (member_access_expression
   name: (name) @property)
 
+(nullsafe_member_access_expression
```

Comment:
Please make them all `@variable.member` instead of `@property` (the ones above too).

Reply:
i dont understand why i should change it for `nullsafe_member_access_expression` when the `member_access_expression` is `@property`

Reply:
You can change all of them, that's the point :)

Reply:
Because `@variable.member` is the correct capture. I may have missed these when I fixed the rest.